superdesk/publish/subscribers.py

```diff
@@ -0,0 +1,215 @@
+            'default': True
+        },
+        'critical_errors': {
+            'type': 'dict',
+            'keyschema': {
+                'type': 'boolean'
+            }
+        },
+        'last_closed': {
+            'type': 'dict',
+            'schema': {
+                'closed_at': {'type': 'datetime'},
+                'closed_by': Resource.rel('users', nullable=True),
+                'message': {'type': 'string'}
+            }
+        },
+        'destinations': {
+            'type': 'list',
+            'required': True,
+            "minlength": 1,
+            'schema': {
+                'type': 'dict',
+                'schema': {
+                    'name': {'type': 'string', 'required': True, 'empty': False},
+                    'format': {'type': 'string', 'required': True},
+                    'delivery_type': {'type': 'string', 'required': True},
+                    'config': {'type': 'dict'}
+                }
+            }
+        },
+        'content_filter': {
+            'type': 'dict',
+            'schema': {
+                'filter_id': Resource.rel('content_filters', nullable=True),
+                'filter_type': {
+                    'type': 'string',
+                    'allowed': ['blocking', 'permitting'],
+                    'default': 'blocking'
+                }
+            },
+            'nullable': True
+        },
+        'global_filters': {
+            'type': 'dict',
+            'keyschema': {
+                'type': 'boolean'
+            }
+        }
+    }
+
+    datasource = {'default_sort': [('_created', -1)]}
+
+    item_methods = ['GET', 'PATCH', 'PUT']
+
+    privileges = {'POST': 'subscribers', 'PATCH': 'subscribers'}
+
+
+class SubscribersService(BaseService):
+    def get(self, req, lookup):
+        if req is None:
+            req = ParsedRequest()
+        if req.args and req.args.get('filter_condition'):
+            filter_condition = json.loads(req.args.get('filter_condition'))
+            return ListCursor(self._get_subscribers_by_filter_condition(filter_condition))
+        return super().get(req=req, lookup=lookup)
+
+    def on_create(self, docs):
+        for doc in docs:
+            self._validate_seq_num_settings(doc)
+
+    def on_update(self, updates, original):
+        self._validate_seq_num_settings(updates)
+
+    def _get_subscribers_by_filter_condition(self, filter_condition):
+        """
+        Searches all subscribers that has a content filter with the given filter condition
+        If filter condition is used in a global filter then it returns all
+        subscribers that not disabled the global filter.
+        :param filter_condition: Filter condition to test
+        :return: List of subscribers
+        """
+        req = ParsedRequest()
+        all_subscribers = list(super().get(req=req, lookup=None))
+        selected_subscribers = {}
+
+        filter_condition_service = get_resource_service('filter_conditions')
```

Comment:
Is there any plan to move 'filter_conditions' and 'content_filters' to core as well?